input_IngestLDDTool_TypeList_1H00_2000.xml

```diff
@@ -0,0 +1,498 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
@lylehuber for this commit, did you mean to move for this IngestLDD file to replace what is at `src/PDS4_CTLI_IngestLDD.xml`?

if you take a look at the branch, this file is at the top level of directory:

https://github.com/pds-data-dictionaries/ldd-ctli/tree/release/1.17.0.0

Also, for consistency purposes, and also to make it a little easier to automate where and what to automatically feed into LDDTool, we have been trying to encourage using the file naming `PDS4_<namespace_id>_IngestLDD.xml`.